src/app/searchResults/searchResults.component.js

```diff
@@ -49,7 +48,17 @@ class SearchResultsController {
     this.searchParams.type = type
 
     if (!this.searchParams.keyword && includes(['ministries', '', undefined], this.searchParams.type)) {
-      this.searchResults = ministries
+      const path = this.$location.$$path
```

Comment:
Not sure what `$location.$$path` is doing. Is it supposed to be `$location.path()`?

Reply:
Fixed